sdk/mixedreality/azure-mixedreality-authentication/src/test/java/com/azure/mixedreality/authentication/MixedRealityStsClientTestBase.java

```diff
@@ -40,13 +45,21 @@ HttpPipeline getHttpPipeline(HttpClient httpClient) {
         final List<HttpPipelinePolicy> policies = new ArrayList<>();
         policies.add(new BearerTokenAuthenticationPolicy(credential, authenticationScope));
 
-        if (getTestMode() == TestMode.RECORD) {
+        if (interceptorManager.isRecordMode()) {
+            List<TestProxySanitizer> customSanitizers = new ArrayList<>();
+            customSanitizers.add(new TestProxySanitizer("$..AccessToken", null, "eyJhbGciOiJub25lIiwidHlwIjoiSldUIn0.eyJlbWFpbCI6IkJvYkBjb250b3NvLmNvbSIsImdpdmVuX25hbWUiOiJCb2IiLCJpc3MiOiJodHRwOi8vRGVmYXVsdC5Jc3N1ZXIuY29tIiwiYXVkIjoiaHR0cDovL0RlZmF1bHQuQXVkaWVuY2UuY29tIiwiaWF0IjoiMTYwNzk3ODY4MyIsIm5iZiI6IjE2MDc5Nzg2ODMiLCJleHAiOiIxNjA3OTc4OTgzIn0.", TestProxySanitizerType.BODY_KEY));
```

Comment:
gave feedback offline; add comment make this more obvious that it's an invalid dummy token